### PR TITLE
changed daemon message

### PR DESCRIPTION
**Pull request by EagleEye25**
changed daemon message for new implimentations in 1.2, for if the daemon disconnects

### CODE REVIEW

File: modules/rpc/rpc.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ exports.call = function(method, params, callback) {
       case 'ECONNREFUSED':
         callback({
           status: 502,
-          message: 'Connection Refused',
+          message: 'Daemon not connected, retrying connection',
```
Comment (vikas-cis) [inline modules/rpc/rpc.js]: Is this change required ?

**Comment by pciavald** [comment]
It seems there's an error without a message somewhere
screenshot_20180108_133656

**Comment by pciavald** [comment]
Where is the second message coming from ? it should not be triggered if daemon is not connected, only if daemon is connected and there's no wallet / popup has been closed.

File: modules/rpc/rpc.js (hunk 1 above)
Comment (pciavald) [reply modules/rpc/rpc.js]: Yeah, I think it better describes the the error to the user as connection is not likely to be actually refused

**Comment by pciavald** [comment]
There's a bug.

- start `particld -rpcuser=test -rpcpassword=lol`
- start wallet

### current

- no error, wallet empty
- wallet quits after some time (logged #625)

### expected

- error: could not connect to daemon
- open small modal to ask the user to ensure his daemon connection details are correct

**Comment by HenkSwardt** [comment]
@EagleEye25 will look into this error scenario shortly.

**Comment by anandsinghparihar** [comment]
@EagleEye25 I just added a sample modal with dummy code (how you can call it etc.. ) now you can work on that, let me know if you need anything  :)

**Comment by vikas-cis** [comment]
@pciavald I have tried the steps that you mention but I am facing issues 
Step : I ran ./particld -rpcuser=test -rpcpassword=lol gives me 
Assertion failed: (false), function SerializationOp, file ./compressor.h, line 219.
Abort trap: 6

**Comment by vikas-cis** [comment]
@HenkSwardt I have discusssed with pierre regarding the issue and tested the via screenshare and it seems all good according to pierre. @pciavald will review and test it again to figure out if he is getting the issue again.

Thanks

**Comment by pciavald** [comment]
I still have this issue.

in the background : 
```
particld -testnet -rpcuser=test -rpcpassword=lol -printtoconsole
```
in the foreground : 
```
npm run start:electron:fast -- -printtoconsole -testnet
```

peek 2018-01-12 22-05

**Comment by pciavald** [comment]
let's get this merged in 1.1 and fix that less common behaviour be more verbose in 1.2
#667

File: src/app/core-ui/main/main-view.component.ts [hunk 2]
```diff
@@ -132,4 +135,9 @@ export class MainViewComponent implements OnInit {
     return sec;
   }
 
+  // Sample code for open modal box
+  openDemonConnectionModal() {
+    const dialogRef = this.dialog.open(DaemonConnectionComponent);
+    dialogRef.componentInstance.text = "Test";
```
Comment (pciavald) [inline src/app/core-ui/main/main-view.component.ts]: fix for linter and/or comment out
